src/app/containers/Timestamp/index.jsx

```diff
@@ -24,16 +39,20 @@ const shortAlphaNumeric = {
   day: date => date.getDate(),
   month: date => months[date.getMonth()],
   year: date => date.getFullYear(),
-  format: (d, m, y) => [d, m, y].join(' '),
+  hour: date => date.getHours(),
```

Comment:
```suggestion
  hour: date => date.getUTCHours(),
```
this will fix the snapshot tests failing

Reply:
**disclaimer** It (inexplicably) makes the `asyncValidateDir helper › should call readScenario for every file in the /data directory` test fail locally, but it should pass remotely.